tsdb/chunkenc/histogram.go

```diff
@@ -227,6 +212,10 @@ func (a *HistogramAppender) GetCounterResetHeader() CounterResetHeader {
 	return CounterResetHeader(a.b.bytes()[2] & 0b11000000)
 }
 
+func (a *HistogramAppender) setCounterResetHeader(cr CounterResetHeader) {
+	a.b.bytes()[2] = (a.b.bytes()[2] & 0b00111111) | (byte(cr) & 0b11000000)
```

Comment:
Maybe worth adding a comment about what this code does. I assume it applies a mask but it is hard to tell which bits it sets.

Reply:
I've replaced 0b11000000 with CounterResetHeaderMask and 0b00111111 with ^CounterResetHeaderMask to make this more obvious and less cryptic everywhere.